atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -799,7 +799,7 @@ private void extractLocalWritesForRow(
                 .get(
                         tableRef,
                         cells,
-                        (table, uncached) -> getInternal(
+                        uncached -> getInternal(
                                 "get", tableRef, uncached, immediateKeyValueService, immediateTransactionService));
```

Comment:
hilariously, the `table` part of the function was never being used (joys of not having `StrictUnusedVariable` checks...), so this technically isn't a change.

Reply:
That was technically a bug that we missed